docs/installation.rst

```diff
@@ -0,0 +1,33 @@
+************
+Installation
+************
+
+Requirements
+============
+
+**astroplan** requires Python 2.7 or 3.3+ (2.6 and 3.2 or earlier are not
+supported) as well as the following packages:
+
+* `Numpy <http://www.numpy.org>`_
+* `Matplotlib <http://www.matplotlib.org>`_
+* `Astropy <http://www.astropy.org>`_
```

Comment:
On Python 3 there's a different Sphinx warning:
https://travis-ci.org/astroplanners/astroplan/jobs/71459795#L870

It's not clear to me if there was a download timeout or a Sphinx warning.
Can you try `python setup.py build_sphinx -l` locally, and if that works, just push a new commit to this branch to trigger another travis-ci build and see if it's fixed?